src/main/java/com/example/CinemaRoom/service/AuthenticationService.java

```
@@ -0,0 +1,16 @@
+package com.example.CinemaRoom.service;
+
+import com.example.CinemaRoom.exception.UnauthorizedException;
+
+public class AuthenticationService {
+
+    private static final String PASSWORD = "super_secret";
+
+    public boolean isValid(String password) {
```

This function should return a boolean value. Please throw an exception where the result of isValid() is utilized.